From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@jongalloway__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jogallow). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

deneha commented:
@jongalloway Thanks for the contribution!

JeffGoldner committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

JeffGoldner committed:
Fix publisher for Chef

yujhongmicrosoft committed:
Merge pull request #3 from MicrosoftDocs/master

g

JeffGoldner committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

ShawnJackson committed:
edit pass: manage-update-multi

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into manage-update-multi

ShawnJackson committed:
edit pass: manage-update-multi

dominicbetts committed:
Update landing page

dominicbetts committed:
Update date

carolinacmoravia committed:
C6881: Hardcoded EN locale in link

Hello, @seguler,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into manage-update-multi

ShawnJackson committed:
edit pass: manage-update-multi

ShawnJackson committed:
fixing merge conflicts

ShawnJackson committed:
edit pass: manage-update-multi

ShawnJackson committed:
Update manage-update-multi.md

v-viksur committed:
(ForumsTeam) Incorrectly mentioned Powershell for Azure Portal procedure  

In the first para of  this documentation we have mentioned PoweShell instead of Azure portal. Requesting you to review and edit.

JeffGoldner committed:
Update the script

JeffGoldner committed:
Update and add new items

JeffGoldner committed:
Fixed an incorrect ISV name

JeffGoldner committed:
Fix order for SQL content

JeffGoldner committed:
Merge branch 'master' into master

JeffGoldner committed:
Add link to previous releases

JeffGoldner committed:
Address issues from PR reviewer

JeffGoldner committed:
address comments

tdykstra committed:
typo

ggailey777 committed:
New tutorial TOC

JeffGoldner committed:
Fix latest round of issues

ggailey777 committed:
Tweak to TOC

JeffGoldner committed:
Fix issues again including bad Notes and duplicate content

Commit:
edit pass: active-directory-passwords-overview

barbkess committed:
loading tutorial

JeffGoldner committed:
More fixes per PR

swikrutibose committed:
(ForumsTeam) Broken link for "Using Windows PowerShell with Resource Manager"

In this document section "Create a resource group for Resource Manager", "Using Windows PowerShell with Resource Manager" link is broken. Requesting you to update this with "https://docs.microsoft.com/en-us/azure/azure-resource-manager/powershell-azure-resource-manager" link.

Commit:
edit pass: active-directory-passwords-reporting

JeffGoldner committed:
fix paint issues

Commit:
november

ggailey777 committed:
Tom's suggestion

Commit:
november

anumjs committed:
Adding analytics tutorial for multitenant database version

anumjs committed:
Updated .png to .PNG

barbkess committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into load

anumjs committed:
Updated .png to .PNG

ggailey777 committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr into s127

anumjs committed:
Implemented acrolinx suggestions

barbkess committed:
replace existing tutorial

anumjs committed:
Removed broken link

christiankuhtz committed:
tweaks to snat exhaust mitigation

ajlam committed:
Merge pull request #1 from MicrosoftDocs/master

Update fork

christiankuhtz committed:
tweaks

barbkess committed:
fix PR remarks

ajlam committed:
Add general guidance

barbkess committed:
fix initial image

barbkess committed:
edits

ShawnJackson committed:
Update active-directory-passwords-policy.md

ajlam committed:
Update object explorer picture

JeffGoldner committed:
Revert "More fixes per PR"

This reverts commit b8dbf9c8614b69b6045b6ef7b1dca69be34fa185.

ajlam committed:
Update TOC

JeffGoldner committed:
try again on these changes

ajlam committed:
Update typos multi-tenant DB provision

yujhongmicrosoft committed:
Uploaded AAD qs

yujhongmicrosoft committed:
Update TOC.md

ShawnJackson committed:
Update active-directory-passwords-reporting.md

spelluru committed:
added details

YuriDio committed:
updating articles

yujhongmicrosoft committed:
Merge branch 'master' into master

spelluru committed:
minor

YuriDio committed:
updating article 1

mimig1 committed:
Cosmos DB: Usability improvements to Java QS.

anumjs committed:
Updated meta title

yujhongmicrosoft committed:
Update documentation-government-aad-auth-qs.md

mimig1 committed:
update

anumjs committed:
Updated headings to lowercase for second word

Commit:
Merge pull request #28262 from christiankuhtz/patch-34

tweaks to snat exhaust mitigation

chipalost committed:
removed text from first image and resized them all

ktoliver committed:
Merge pull request #28152 from ggailey777/s127

New tutorial TOC

yujhongmicrosoft committed:
Update documentation-government-aad-auth-qs.md

ajlam committed:
Update wording

ajlam committed:
Delete MultiTenantCatalog.png

ajlam committed:
Update multitenant catalog pic

ShawnJackson committed:
Update active-directory-passwords-reporting.md

ajlam committed:
Update image link

JeffGoldner committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

ajlam committed:
Update wording

chipalost committed:
made the intro sections match between the Linux and Windows versions

christiankuhtz committed:
tweak

JeffGoldner committed:
Fix TOC again

barbkess committed:
fix name

ShawnJackson committed:
Update active-directory-passwords-reporting.md

GitHubber17 committed:
Merge pull request #28201 from barbkess/load

loading tutorial

Commit:
november

Commit:
Merge pull request #28290 from christiankuhtz/patch-35

tweak

ShawnJackson committed:
Merge pull request #28187 from LoriMerrick/three-active-directory-password

edit pass: three-active-directory-passwords

Commit:
Add more parameter info from Verizon

barbkess committed:
fix-ups

GitHubber17 committed:
Merge pull request #28283 from YuriDio/UpdatesNov_02

updating articles

gsacavdm committed:
Update gov marketplace article.

PRMerger12 committed:
Merge pull request #27459 from carolinacmoravia/patch-452

C6881: Hardcoded EN locale in link

GitHubber17 committed:
Merge pull request #28254 from anumjs/master

Added analytics tutorial for multitenantdb version

barbkess committed:
acrolinx

cherylmc committed:
Update vpn-gateway-howto-vnet-vnet-resource-manager-portal.md

barbkess committed:
acrolinx

Commit:
november

Commit:
Merge pull request #28304 from cherylmc/patch-2

Update vpn-gateway-howto-vnet-vnet-resource-manager-portal.md

spelluru committed:
added verify output section

JasonWHowell committed:
Update concepts-compatibility.md

christiankuhtz committed:
tweaks

Commit:
format update

PRMerger13 committed:
Merge pull request #28310 from JasonWHowell/patch-1

Update concepts-compatibility.md

gsacavdm committed:
Minor fixes to azure gov marketplace doc

ktoliver committed:
Merge pull request #28289 from chipalost/removeTextFromImages

removed text from first image and resized them all

ktoliver committed:
Merge pull request #28282 from spelluru/adfqsdotnet1120

added details

ktoliver committed:
Merge pull request #28285 from mimig1/11202017usability

Cosmos DB: Usability improvements to Java QS.

Commit:
Update iot-hub-python-getstarted.md

ktoliver committed:
Merge pull request #28298 from barbkess/load

Load

ktoliver committed:
Merge pull request #28292 from JeffGoldner/master

Updates for Azure Stack SQL, MySQL and Marketplace

ktoliver committed:
Merge pull request #28299 from gsacavdm/patch-2

Update gov marketplace article.

mayanknayar committed:
Site recovery articles refresh

Freshness update for site recovery articles

mayanknayar committed:
Troubleshooting corrections

Minor corrections in troubleshooting articles

mayanknayar committed:
A2A Troubleshooting update

Minor typo corrections

linda33wj committed:
Add tips for format settings

linda33wj committed:
Refresh V1 perf content and ref table

PRMerger16 committed:
Merge pull request #28243 from swikrutibose/patch-2

(ForumsTeam) Broken link for "Using Windows PowerShell with Resource …

Commit:
Enhancing customer onboarding experience

Based on the customer feedback the part of the page pertaining to onboarding, that is using the Intelligent Insights was not clear. The purpose of this edit is to clarify onboarding steps. Also included embedded video with some hands-on examples.

PRMerger16 committed:
Merge pull request #28013 from v-viksur/patch-11

(ForumsTeam) Incorrectly mentioned Powershell for Azure Portal procedure

Commit:
Update sql-database-intelligent-insights.md

gsacavdm committed:
Replace AAD acronycm

rjagiewich committed:
Merge pull request #27434 from dominicbetts/suite-landing-page

Update IoT Suite landing page links

Commit:
november

PRMerger16 committed:
Merge pull request #28300 from dksimpson/dean-380

Add more parameter info from Verizon

MikeDodaro committed:
fix endpoint URL

yujhongmicrosoft committed:
Update TOC.md

yujhongmicrosoft committed:
Update documentation-government-aad-auth-qs.md

yujhongmicrosoft committed:
Update documentation-government-aad-auth-qs.md

barclayn committed:
updating articles

yujhongmicrosoft committed:
Update documentation-government-aad-auth-qs.md

ajlam committed:
Fix issues from review.

barclayn committed:
making date changes

MikeDodaro committed:
Modify logic for search and category

ajlam committed:
Fix blocking issue

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo

YuriDio committed:
updating articles

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

yujhongmicrosoft committed:
Update documentation-government-aad-auth-qs.md

PRMerger19 committed:
Merge pull request #28314 from hning86/master

format update

saghorpa committed:
link added for blog

barclayn committed:
updating dates

barclayn committed:
updating dates

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPublicRepo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

barclayn committed:
correcting errors

jennyhunter-msft committed:
Remove Bash as runbook type

- Removed Bash as a runbook type (This is not supported and shouldn't have been added).

PRMerger12 committed:
Merge pull request #28312 from christiankuhtz/patch-36

tweaks

barclayn committed:
making changes based on Acrolynx plugin recommendations

jennyhunter-msft committed:
Minor grammar fixes

- Minor grammar fixes

vladvino committed:
Update api-management-advanced-policies.md

barclayn committed:
updating dates

JasonWHowell committed:
Update time-series-insights-parameterized-urls.md

JasonWHowell committed:
Update time-series-insights-diagnose-and-solve-problems.md

vladvino committed:
Update api-management-policy-expressions.md

JasonWHowell committed:
Update time-series-insights-environment-mitigate-latency.md

JasonWHowell committed:
Update time-series-insights-manage-reference-data-csharp.md

JasonWHowell committed:
Update time-series-insights-how-to-add-an-event-source-iothub.md

JasonWHowell committed:
Update time-series-insights-authentication-and-authorization.md

JasonWHowell committed:
Update time-series-insights-how-to-add-an-event-source-eventhub.md

JasonWHowell committed:
Update time-series-insights-parameterized-urls.md

Commit:
Fixed the inconsistent references to the null value.

Commit:
Merge pull request #28320 from shizn/patch-7

Update iot-hub-python-getstarted.md

PRMerger18 committed:
Merge pull request #28363 from gsacavdm/patch-2

Replace AAD acronycm

linda33wj committed:
Fix broken link

linda33wj committed:
Add Salesforce Service Cloud support

PRMerger19 committed:
Merge pull request #28404 from vladvino/patch-9

Update api-management-policy-expressions.md

PRMerger17 committed:
Merge pull request #28394 from vladvino/patch-8

Update api-management-advanced-policies.md

PRMerger13 committed:
Merge pull request #28392 from jennyhunter-msft/patch-2

Remove Bash as runbook type

MikeDodaro committed:
Upgrade application code

PRMerger17 committed:
Merge pull request #28409 from JasonWHowell/patch-8

TSI: Update time-series-insights-how-to-add-an-event-source-eventhub.md

linda33wj committed:
Add Salesforce Service Cloud support

PRMerger19 committed:
Merge pull request #28408 from JasonWHowell/patch-7

TSI: Update time-series-insights-authentication-and-authorization.md

PRMerger19 committed:
Merge pull request #28407 from JasonWHowell/patch-6

TSI: Update time-series-insights-how-to-add-an-event-source-iothub.md

linda33wj committed:
Add Salesforce Service Cloud support

PRMerger12 committed:
Merge pull request #28406 from JasonWHowell/patch-5

TSI: Update time-series-insights-manage-reference-data-csharp.md

PRMerger13 committed:
Merge pull request #28405 from JasonWHowell/patch-4

TSI: Update time-series-insights-environment-mitigate-latency.md

Commit:
Merge pull request #28402 from JasonWHowell/patch-3

TSI: Update time-series-insights-diagnose-and-solve-problems.md

PRMerger15 committed:
Merge pull request #28401 from JasonWHowell/patch-2

TSI: Update time-series-insights-parameterized-urls.md

cychua committed:
Add information about proxy timeout

Commit:
Merge pull request #28424 from cychua/master

Add information about proxy timeout

cychua committed:
Adding no support for disjoint ad domains

cychua committed:
Update how staging affects PHS

YuriDio committed:
updating articles 2

PRMerger17 committed:
Merge pull request #28429 from cychua/master

Adding no support for disjoint ad domains

cychua committed:
Add clarification on Group filter in multi forest

cychua committed:
IPv6

Commit:
Added Intelligent Insights how-to video

In the section for Intelligent Insights, I've added a hands-on video as we are trying to drive the product adoption. Customer feedback was they needed more hands-on in onboarding / getting started with the product. Thanks.

Commit:
Update log-analytics-azure-sql.md

PRMerger15 committed:
Merge pull request #28432 from cychua/master

Add clarification on Group filter in multi forest

PRMerger10 committed:
Merge pull request #28433 from danimir/patch-6

Added Azure SQL Analytics solution how-to video

PRMerger15 committed:
Merge pull request #28361 from barclayn/date-update

updating articles

cephalin committed:
added livefyre people

cephalin committed:
open link in tab

PRMerger20 committed:
Merge pull request #28436 from cephalin/webjob

added livefyre people

daveirwin1 committed:
Removed mentions of dependency-agent-connector.

PRMerger14 committed:
Merge pull request #28437 from cephalin/link

open link in tab

ponatara committed:
Update site-recovery-support-matrix-to-azure.md

ponatara committed:
Update site-recovery-support-matrix-azure-to-azure.md

PRMerger20 committed:
Merge pull request #28438 from daveirwin1/master

Removed mentions of dependency-agent-connector.

GitHubber17 committed:
Merge pull request #28252 from MarkusVi/november

november

ktoliver committed:
Merge pull request #28340 from danimir/patch-5

Enhancing customer onboarding experience

ktoliver committed:
Merge pull request #28367 from YuriDio/UpdatesNov_03

updating articles

TomShinder committed:
Update governance-in-azure.md

ktoliver committed:
Merge pull request #28336 from linda33wj/master

Update ADF copy content

Commit:
november21

Lisaco88 committed:
Merge pull request #28280 from yujhongmicrosoft/master

TOC changes and aad auth qs

tdykstra committed:
fix internal anchor

PRMerger13 committed:
Merge pull request #28449 from TomShinder/patch-41

Update governance-in-azure.md

v-alje committed:
Merge pull request #28368 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #28352 from MikeDodaro/fix-news-endpoint-tutorial

Fix endpoint URL in tutorial

Commit:
fixed broken or incorrect links

PRMerger12 committed:
Merge pull request #28450 from MarkusVi/november21

november21

PRMerger15 committed:
Merge pull request #28124 from tdykstra/patch-8

typo

PRMerger14 committed:
Merge pull request #28451 from tdykstra/patch-9

fix internal anchor

PRMerger20 committed:
Merge pull request #28439 from ponatara/patch-4

Update site-recovery-support-matrix-to-azure.md

Commit:
Merge pull request #28441 from ponatara/patch-5

Update site-recovery-support-matrix-azure-to-azure.md

Ja-Dunn committed:
Merge pull request #27795 from ShawnJackson/manage-update-multi

edit pass: manage-update-multi

PRMerger16 committed:
Merge pull request #28398 from saghorpa/SUSELinkUpdate11212017

link added for blog

PRMerger17 committed:
Merge pull request #28452 from craigcaseyMSFT/v-craic1122

fixed broken or incorrect links

Lisaco88 committed:
Merge pull request #28273 from ajlam/master

Add Wingtip Tickets General Guidance

Lisaco88 committed:
Merge pull request #28278 from ajlam/patch-2

Update typos multi-tenant DB provision

PRMerger20 committed:
Merge pull request #28410 from piotrci/DGDocNullUpdate

Fixed the inconsistent references to the null value.

Lisaco88 committed:
Merge pull request #28333 from mayanknayar/master

Site recovery articles refresh

GitHubber17 committed:
Merge pull request #28395 from barclayn/updates-4

updating dates

GitHubber17 committed:
Merge pull request #28366 from barclayn/updates-2

making date changes

Lisaco88 committed:
Merge pull request #28383 from barclayn/updates-3

Updates 3

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
